client/scripts/views/pages/stats.ts

```diff
@@ -127,10 +128,23 @@ const StatsPage: m.Component<{}, { requested: boolean, error: string, data }> =
       ],
     });
 
+    const isAdmin =
+        app.user.isSiteAdmin ||
+        app.user.isAdminOfEntity({
+          chain: app.activeChainId(),
+          community: app.activeCommunityId(),
+        });
+    const isMod = app.user.isRoleOfCommunity({
+      role: 'moderator',
+      chain: app.activeChainId(),
+      community: app.activeCommunityId(),
+    });
+
```

Comment:
Remove this (like discussed above) and other instance in which you see it before the component.

Reply:
Cool! Will fix these things.